Client/utils/appAPI.js

```diff
@@ -0,0 +1,273 @@
+import axios from 'axios';
+
+// const AppActions = require('../actions/AppActions');
+import AppActions from '../actions/AppActions';
+
+module.exports = {
+  registerNewUser(user) {
+    console.log(user);
```

Comment:
Console logs are good for debugging during development but shouldn't be pushed to production.